### DISCUSSION TITLE
Remove all VSCode folding rules and keep only ExplicitFolding ones

### DISCUSSION THREAD

**Discussion by fsoft72** [zokugun/vscode-explicit-folding#66] [Q&A]
Hello, is it possible to remove all VSCode folding rules and keep only the ones defined with ExplicitFolding?

My problem is that sometimes, when I click the hotkey for folding, it folds based on cursor position and folds only some rows of the whole text because it stops before reaching the ExplicitFolding defined rules.

**Comment by daiyam** [comment]
No, VSCode don't allow that possibility. But, I've made a custom build ([MrCode](https://github.com/zokugun/MrCode)) for mainly that reason...
See [editor.foldingStrategy](https://github.com/zokugun/MrCode/blob/master/docs/settings/editor.foldingStrategy.md) for more details

**Comment by TonyGravagno** [comment]
I've been thinking about this too. I don't like that we can't completely disable VSCode defaults - I can't tell if I'm doing something wrong with this extension or if the IDE is doing something "for" me.

I'm not anxious to migrate to Mr Code or another platform for this one feature. Is there anything new on this?
Thanks!

**Reply by daiyam** [reply]
Now, using the settings:
```
"editor.foldingStrategy": "auto",
"editor.defaultFoldingRangeProvider": "zokugun.explicit-folding",
```

You should be fine.

I did removed the `editor.foldingStrategy` setting from `MrCode`.
